pandas/io/formats/format.py

```diff
@@ -1897,7 +1897,7 @@ def _format(x):
 
         vals = self.values
         if isinstance(vals, Index):
-            vals = vals._values
+            vals = vals._ndarray_values
```

Comment:
Currently, an extension array will end up here (using the GenericArrayFormatter), so should this rather be `np.array(vals._values)` ? 
(the implementation of `_ndarray_values` on an ExtensionArray should not influence its formatting?)

Reply:
It seems like by this point we always have a regular `Index`, not an extension array, so it doesn't matter in terms of output. Changing to `_values` since that's clearer.